eth/gen_config.go

```diff
@@ -82,6 +86,8 @@ func (c Config) MarshalTOML() (interface{}, error) {
 	enc.RPCGasCap = c.RPCGasCap
 	enc.Checkpoint = c.Checkpoint
 	enc.CheckpointOracle = c.CheckpointOracle
+	enc.OverrideIstanbul = c.OverrideIstanbul
+	enc.OverrideMuirGlacier = c.OverrideMuirGlacier
```

Comment:
As far as I can recall, we don't toml:ize the overrides, because they should only be applied when explicitly called for, not "accidentally" remembered by a config file

Reply:
In that case you need to add `toml:"-"` next time. I will send a PR to remove them after this PR since they're not needed anymore either way (comment says "delete after the fork").